frontend/app/services/SubscriptionService.scala

```diff
@@ -131,12 +133,29 @@ object SubscriptionService {
   }
 }
 
-class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
-                          val zuoraSoapService: ZuoraSoapService,
+class SubscriptionService(val zuoraSoapService: ZuoraSoapService,
                           val zuoraRestService: ZuoraRestService) extends AmendSubscription with LazyLogging {
 
   import SubscriptionService._
-  val MembershipProductType = "Membership"
```

Comment:
Two remarks. First, this a member of a class, not an object, hence it's not a constant it's a immutable value within the class. Second, only the first letter of constant names should be capitalised. All caps statics are so Java! ;)